test/TestExpectations.json

```diff
@@ -3366,7 +3372,7 @@
     "testIdPattern": "[target.spec] Target Browser.waitForTarget should wait for a target",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["chrome", "webDriverBiDi"],
-    "expectations": ["PASS"]
+    "expectations": ["FAIL", "PASS"]
```

Comment:
Why is this failing?

Reply:
See https://github.com/puppeteer/puppeteer/issues/11749 and chat.